onnxruntime/python/tools/transformers/run_benchmark.sh

```diff
@@ -88,7 +88,7 @@ if [ "$run_install" = true ] ; then
   else
     pip install onnxruntime-gpu
   fi
-  pip install --upgrade onnx coloredlogs packaging psutil py3nvml onnxconverter_common numpy transformers
+  pip install --upgrade onnx coloredlogs packaging psutil py3nvml onnxconverter_common numpy transformers sympy
```

Comment:
I suggest you put all the packages in a separated file named requirements.txt, so that we can use programs to track what packages are used by this product.

Reply:
sympy is in https://github.com/microsoft/onnxruntime/blob/master/requirements-dev.txt. 

This line contains some packages required for running the script.